packages/page-accounts/src/modals/ProxyOverview.tsx

```diff
@@ -96,7 +96,7 @@ function PrevProxy ({ index, onRemove, typeOpts, value: [accountId, type] }: Pre
   );
 }
 
-function NewProxy ({ index, onChangeAccount, onChangeType, onRemove, proxiedAccount, typeOpts, value: [accountId, type] }: NewProxyProps): React.ReactElement<NewProxyProps> {
+function NewProxy({ index, onChangeAccount, onChangeType, onRemove, proxiedAccount, typeOpts, value: [accountId, type] }: NewProxyProps): React.ReactElement<NewProxyProps> {
```

Comment:
```suggestion
function NewProxy ({ index, onChangeAccount, onChangeType, onRemove, proxiedAccount, typeOpts, value: [accountId, type] }: NewProxyProps): React.ReactElement<NewProxyProps> {
```

Fix, CI fails linting due to this.